lib/di/global/get_it_initializer.dart

```diff
@@ -264,5 +265,8 @@ class GetItInitializer {
     getIt.registerSingleton<UpdateGroupChatInteractor>(
       UpdateGroupChatInteractor(),
     );
+    getIt.registerLazySingleton<UpdatePinnedMessagesInteractor>(
+      () => UpdatePinnedMessagesInteractor(),
+    );
```

Comment:
why we need it as a singleton?

Reply:
when you pin a message, if pin have not done, you back to chat screen, so this interactor need to singleton to continue execute